extensions/binding/org.eclipse.smarthome.binding.hue.test/src/test/groovy/org/eclipse/smarthome/binding/hue/test/HueLightHandlerOSGiTest.groovy

```diff
@@ -428,7 +426,7 @@ class HueLightHandlerOSGiTest extends OSGiTest {
             assertBridgeOnline(hueLightHandler.getBridge())
             enableHueLightChannels(hueLight)
             //Run the initialize method in case of PAR16 50 TW bulb testcase
-            if(currentState.modelId == 'PAR16 50 TW') {
+            if(currentState.modelId.equals(OSRAM_PAR16_MODELID)) {
```

Comment:
FTR: This code has been meanwhile fixed by https://github.com/eclipse/smarthome/pull/687.